backend/LexBoxApi/Controllers/FwLiteReleaseController.cs

```diff
@@ -11,17 +13,22 @@ namespace LexBoxApi.Controllers;
 [ApiController]
 [Route("/api/fwlite-release")]
 [ApiExplorerSettings(GroupName = LexBoxKernel.OpenApiPublicDocumentName)]
-public class FwLiteReleaseController(IHttpClientFactory factory, HybridCache cache) : ControllerBase
+public class FwLiteReleaseController(FwLiteReleaseService releaseService) : ControllerBase
 {
-    private const string GithubLatestRelease = "GithubLatestRelease";
 
     [HttpGet("download-latest")]
     [AllowAnonymous]
     [ProducesResponseType(StatusCodes.Status404NotFound)]
-    public async Task<ActionResult> LatestDownload()
+    public async Task<ActionResult> DownloadLatest()
     {
-        var latestRelease = await GetLatestRelease(default);
-        if (latestRelease is null) return NotFound();
+        using var activity = LexBoxActivitySource.Get().StartActivity();
+        var latestRelease = await releaseService.GetLatestRelease(default);
```

Comment:
eh? not really, it doesn't really matter in this context, I just exposed the argument since it was used by Hybrid cache, but it doesn't really make much sense for our use case, so I'll put default on the method so the caller doesn't have to think about it